test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/RandomGeneratorTest.java

```diff
@@ -276,8 +277,41 @@ public void shouldAlwaysPickSameElementFromAnEnumWithSingleElement() {
         final Generator<SingleEnum> enumGenerator = randomEnum(SingleEnum.class);
 
         // then
-        typeCheck(enumGenerator, s -> enumGenerator.next().compareTo(enumGenerator.next()) == 0).verify(times(NUMBER_OF_TIMES));
+        typeCheck(enumGenerator, s -> enumGenerator.next().compareTo(enumGenerator.next()) == 0)
+                        .verify(times(NUMBER_OF_TIMES));
     }
 
-    enum SingleEnum {SINGLE_VALUE}
-}
\ No newline at end of file
+    @Test
+    public void shouldReturnAnInstanceEmailAddressGenerator() {
```

Comment:
Don't see much value in these tests. I think testing EMAIL_ADDRESS that it generates valid email address would naturally test that we get an instance of EMAIL_ADDRESS object!. Same applies for below tests.